packages/backend-proxy-middleware/src/base/proxy.ts

```diff
@@ -40,7 +40,7 @@ export const ProxyEventHandlers = {
      */
     onProxyReq(proxyReq: ClientRequest, _req?: IncomingMessage, _res?: ServerResponse, _options?: ServerOptions) {
         if (proxyReq.path?.includes('Fiorilaunchpad.html') && !proxyReq.headersSent) {
-            proxyReq.setHeader('accept-encoding', 'br');
+            proxyReq.setHeader('accept-encoding', '*');
```

Comment:
The previous code had this specific `accept-encoding` header, and now you change it to accept everything. That makes me wonder, if you need it at all, i.e. would it also work if you remove the complete if block?

Reply:
@tobiasqueck good point - I had tried removing the whole block, but ran into the same encoding error `net::ERR_CONTENT_DECODING_FAILED 200` if it wasn't set at all. Seems like it has to be set.